### ISSUE TITLE
Is LibWebSockets product ready?

### ISSUE THREAD

**Issue by WenchaoXia** [feixuwu/UEWebsocket#67]
I am integerating it into UE4 also as part of RPC, but I found the API is very unfriendly like a low lever driver, did you used it in your product?

**Comment by feixuwu** [comment]
yes, there's a lot of people use it in product, this is not rpc, if u need it, u can extend it on websocket.

获取 Outlook for Android<https://aka.ms/ghei36>
________________________________
